rise-google-sheet.html

```diff
@@ -76,7 +76,7 @@
 </dom-module>
 
 <!-- build:version -->
-<script>var sheetVersion = "2.3.0";</script>
+<script>var sheetVersion = "2.4.1";</script>
```

Comment:
@donnapep It does, but in order for it to be accurate, a `gulp build` must be run after we've changed the version in package.json. This was last released with an incorrect version because I changed the version in package.json right before pushing the branch and didn't run a build.